From 837ac13b5fba0a56f09e1c6f26e7eb9d945c7860 Mon Sep 17 00:00:00 2001
From: Maxr1998 <max.rumpf1998@gmail.com>
Date: Thu, 29 Dec 2016 15:20:39 +0100
Subject: [PATCH] Patch Makefile

---
 Makefile | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/Makefile b/Makefile
index 5a21744..2ae808b 100644
--- a/Makefile
+++ b/Makefile
@@ -779,6 +779,24 @@ ifeq ($(shell $(CONFIG_SHELL) $(srctree)/scripts/gcc-goto.sh $(CC)), y)
 	KBUILD_CFLAGS += -DCC_HAVE_ASM_GOTO
 endif
 
+# +++ ASUS_BSP : miniporting : add ASUS software version support
+ifneq ($(BUILD_NUMBER),)
+        KBUILD_CPPFLAGS += -DASUS_SW_VER=\"$(BUILD_NUMBER)\"
+else
+        KBUILD_CPPFLAGS += -DASUS_SW_VER=\"WI501Q_ENG\"
+endif
+# --- ASUS_BSP : miniporting : jackson : add ASUS software version support
+KBUILD_CPPFLAGS += -DASUS_DEBUG=2
+# +++ASUS_BSP: "support mutliple project build"
+KBUILD_CPPFLAGS += -DASUS_WI501Q_PROJECT=1
+#--- ASUS_BSP: "support mutliple project build"
+
+# +++ ASUS_BSP: "factory compile option support"
+ifneq ($(ASUS_FACTORY_BUILD),)
+        KBUILD_CPPFLAGS += -DASUS_FACTORY_BUILD=1
+endif
+# --- ASUS_BSP: "factory compile option support"
+
 include $(srctree)/scripts/Makefile.kasan
 include $(srctree)/scripts/Makefile.extrawarn
 include $(srctree)/scripts/Makefile.ubsan
-- 
2.11.0

